README.md

```diff
@@ -500,7 +503,8 @@ response.card({
 ## Error Handling
 
 Handler functions should not throw exceptions. Ideally, you should catch errors in your handlers using try/catch and respond with an appropriate output to the user. If exceptions do leak out of handlers, they will be thrown by default. Any exceptions can be handled by a generic error handler which you can define for your app. Error handlers cannot be asynchronous.
-
+The generic error handler, below, will automatically resolve the promise (request) with `response.send()` if it does not occur in the error handler.
+`response.fail(message, exception)` should only be called from a controlled calling environment, e.g. your own server, not AWS Lambda.
```

Comment:
Why would this be the case? You should be able to use `response.fail` to force a failure response in any environment. Particularly this will trigger `.error` and `.post`.